tests/make/collect_tests.py

```diff
@@ -18,6 +18,8 @@ def collect(path):
         if not SAFE_PATH.match(filename):
             print(f"skipping {filename!r}, use only [a-zA-Z0-9_./] in filenames")
             continue
+        if entry.name.startswith("skip_"):
```

Comment:
Should we also print a message that we are skipping the test like above?

Reply:
I decided to not print anything in this case as this reruns on basically every change to the tests, so I only print stuff that is likely to be an error.